javascript/ql/lib/semmle/javascript/PackageExports.qll

```diff
@@ -141,6 +141,19 @@ private DataFlow::Node getAValueExportedByPackage() {
     result = unique( | | call.getCalleeNode().getAFunctionValue()).getAReturn()
   )
   or
+  // the exported value is a function that returns another import.
```

Comment:
Do you know if this pattern is common in this syntactic form? If so, it would probably be worth it to support this kind of reexport directly in our import graph as well.

Reply:
I'm not sure if we can put this kind of "reexport" directly into the import graph.  

At the import site it requires a user to write something like:  
```JavaScript
var lib = require("lib")(conf);
```
So it's only "reexported" when the client calls the function.